src/components/Sidebar/NewSidebar.tsx

```diff
@@ -1,3 +1,35 @@
+import { useUser } from "@dashboard/auth";
+import { Box, List, Text } from "@saleor/macaw-ui/next";
 import React from "react";
+import { useIntl } from "react-intl";
+import { Link } from "react-router-dom";
 
-export const NewSidebar = () => <div>Work in progress</div>;
+import useMenuStructure from "./useMenuStructure";
+
+export const NewSidebar = () => {
+  const intl = useIntl();
+  const { user } = useUser();
+  const [menuStructure] = useMenuStructure(intl, user);
```

Comment:
suggestion: why not in this way
```suggestion
  const menuStructure = useMenuStructure(intl, user);
```